### ISSUE TITLE
Cannot find module fetchStatamicResource

### ISSUE THREAD

**Issue by r6203** [robertguss/gatsby-source-statamic#10]
I'm facing an issue while trying the plugin. I've just installed the plugin and configured it in my "gatsby-config.js" (no queries, yet).

```
Error in "/home/work/sites/gatsby-simplefolio/node_modules/gatsby-source-statamic/gatsby-node.js": Cannot find module './helpers/fetchStatamicResource'
Require stack:
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby-source-statamic/gatsby-node.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/bootstrap/resolve-module-exports.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/bootstrap/load-plugins/validate.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/bootstrap/load-plugins/load.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/bootstrap/load-plugins/index.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/services/initialize.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/services/index.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/state-machines/develop/services.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/state-machines/develop/index.js
- /home/work/sites/gatsby-simplefolio/node_modules/gatsby/dist/commands/develop-process.js
- /home/work/sites/gatsby-simplefolio/.cache/tmp-423618-1baeVdlzTP9W
```

Oddly enough, I couldn't find the "fetchStatamicResource.js" file in this repo which makes me wonder how the plugin has ever worked.

Can someone help me? I'm feeling dumb. :sweat_smile:

**Comment by robertguss** [comment]
I am not sure what the issue is as this stack trace is not helpful. That file does not exist in the plugin so I am not sure where that reference is even coming from. Can you provide more context or more details on what you are trying to do?

**Comment by robertguss** [comment]
hey @r6203 I have no idea how that file is missing. I am fixing it now.

**Comment by robertguss** [comment]
@r6203 I am so sorry but this plugin was seriously broken. I have no idea how or why so many of the helper files were missing. Thank you so much for brining this to my attention. I just published a new version of it on npm 0.1.3. I have tested everything locally on my end and it seems to be working again. Please let me know if you have any other issues and thanks again :)

https://www.npmjs.com/package/gatsby-source-statamic

**Comment by r6203** [comment]
Thank you, Robert for the fast fix. :)
